state/src/interface/matching_pools.rs

```diff
@@ -68,55 +68,4 @@ impl State {
     ) -> Result<ProposalWaiter, StateError> {
         self.send_proposal(StateTransition::AssignOrderToMatchingPool { order_id, pool_name }).await
     }
-
-    /// Create the global matching pool
-    pub async fn create_global_matching_pool(&self) -> Result<ProposalWaiter, StateError> {
-        self.create_matching_pool(GLOBAL_MATCHING_POOL.to_string()).await
-    }
-
-    /// Assign an order to the global matching pool
-    pub async fn assign_order_to_global_matching_pool(
-        &self,
-        order_id: OrderIdentifier,
-    ) -> Result<ProposalWaiter, StateError> {
-        self.assign_order_to_matching_pool(order_id, GLOBAL_MATCHING_POOL.to_string()).await
-    }
```

Comment:
I've removed these two methods from the state interface because we effectively never use them. We create the global matching pool & assign orders to it directly via the `tx` (the latter is in the next PR)